server/rpc.go

```diff
@@ -50,6 +51,88 @@ type Broadcaster interface {
 	Sign([]byte) ([]byte, error)
 }
 
+type BroadcastSessionsManager struct {
```

Comment:
We don't currently use this struct within `rpc.go` , should we put it somewhere else? I'm thinking either broadcast.go, mediaserver.go or its own file.

Reply:
hmm don't we use it in `NewSessionManager`, and have associated `BroadcastSessionsManager` methods in the same file - `selectSession`, `removeSession`, etc? Or, do you mean something else?

Reply:
We don't actually call any of this in `rpc.go` though, so I'm not sure that this really needs to live there.

Reply:
`NewSessionManager` could go in mediaserver.go